Assets/HotmaxEditorVR/builder/uiClasses/builderMenuManager.cs

```diff
@@ -15,6 +16,8 @@ public class builderMenuManager : MonoBehaviour
 
     void Start()
     {
+        _stateManagerMutatorRef = init._stateManagerMutatorRef;
```

Comment:
i think this is the only place where you don't use FindObjectofType(StateManagerMutatorRef)..
any reason?